### PR TITLE
Extending CronJob

### PR DESCRIPTION
**Pull request by ljakimczuk**
Discussion: https://gigantic.slack.com/archives/C02GDJJ68Q1/p1645696632194679

### CODE REVIEW

**Review by rossf7 — APPROVED** [review]
LGTM

File: manifests/giantswarm/additional-testing-resources/resume-collection-kustomization.yaml [hunk 1]
```diff
@@ -11,12 +11,13 @@ spec:
...
-                - collection
-                - --namespace
-                - flux-giantswarm
+            - command:
+                - sh
+                - -c
+                - |
+                  FLAGS="--namespace flux-giantswarm --verbose"
+                  flux resume kustomization collection $FLAGS
+                  flux resume kustomization collection $FLAGS
```
Comment (rossf7) [inline manifests/giantswarm/additional-testing-resources/resume-collection-kustomization.yaml:20]:
```suggestion
                  flux resume kustomization flux $FLAGS
```

@ljakimczuk Actually shouldn't this be the flux kustomization?

Comment (ljakimczuk) [reply manifests/giantswarm/additional-testing-resources/resume-collection-kustomization.yaml:20]: ah, of course it should, stupid mistake, thanks @rossf7 😄

**Review by rossf7 — APPROVED** [review]
LGTM

**Pull request merged**